src/encoded/schemas/publication.json

```diff
@@ -171,11 +171,16 @@
         }
     },
     "columns": {
+        "nonexistent_year_field" : {
+            "title" : "Year",
+            "notes": "We use date_published here but only render out year."
+        },
         "journal": {
             "title": "Journal"
         },
         "date_published": {
-            "title": "Publication Date"
+            "title": "Publication Date",
+            "default_hidden" : true
```

Comment:
Show month + year. Maybe could remove this completely and/or add other columns; idk.

Reply:
Wait, if you have the date_published, can't you parse that on the front end to get the year? Why does this 'nonexistent_year_field' need to be in the schema?

Reply:
Is just a dummy field under `columns` and isn't a real field in DB (or anywhere else in backend) and just used on frontend as dummy field on "column" schema. Which realizing presents an issue when click "sort" .. thx for the catch!

Yeah agreed. Originally wanted to preserve `date_published` as its own column figuring that it'd have much more fine-grained info incl. day+month, but it seems to usually only have month so seems is less useful than thought originally.